packages/collector/test/tracing/messaging/node-rdkafka/test.js

```diff
@@ -564,7 +571,16 @@ function verifyResponseAndMessage(response, consumerControls, withError, objectM
   })[0];
 
   expect(message).to.exist;
-  expect(Buffer.from(message.value.data, 'utf8').toString()).to.equal('Node rdkafka is great!');
+  const messagePayload = Buffer.from(message.value.data, 'utf8').toString();
+  expect(messagePayload).to.equal('Node rdkafka is great!');
+
+  const headerNames = [];
+  message.headers.forEach(keyValuePair => {
+    const key = Object.keys(keyValuePair)[0];
+    headerNames.push(key);
+  });
+  allInstanaHeaders.forEach(headerName => expect(headerNames).to.not.contain(headerName));
```

Comment:
Just adding a small verification step here that we indeed removed all the Instana related headers.